packages/expo-crypto/ios/CryptoModule.swift

```diff
@@ -16,6 +16,8 @@ public class CryptoModule: Module {
     Function("getRandomBase64String", getRandomBase64String)
 
     Function("getRandomValues", getRandomValues)
+
+    Function("randomUUID", randomUUID)
```

Comment:
Since these functions are one-liners, the closure syntax is probably better. Actually I would always prefer that, just try to keep them short
```suggestion
    Function("randomUUID") {
      return UUID().uuidString
    }
```